### ISSUE TITLE
Layers break after styling and unusable for other stories

### ISSUE THREAD

**Issue by reynalddizon** [MapStory/mapstory#605]
To replicate this issue,

I imported: https://mapstory.org/layers/geonode:0406a_cables
and started composing a story with it (one chapter, one storylayer, one storybox and one storypin).

The first mapstory is: 0406g Cables. https://mapstory.org/story/24297

1. I added the storylayer successfully. No errors.
<img width="1280" alt="screen shot 2017-04-06 at 4 43 18 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763392/226099e2-1ae9-11e7-852e-c6cc0d612309.png">

2. Entered the styling interface. Layer style changed to the default black line style.
<img width="1280" alt="screen shot 2017-04-06 at 4 43 30 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763419/3a78cb76-1ae9-11e7-8e7f-872d4c835c84.png">

3. Successfully changed the style to graduated. 
<img width="1280" alt="screen shot 2017-04-06 at 4 43 51 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763444/558b25bc-1ae9-11e7-98f4-8cee7ae1fc9e.png">

4. Changed my mind and switched back to simple. Failed. The map still displays graduated style but the style interface is in simple with red color.
<img width="1280" alt="screen shot 2017-04-06 at 4 45 29 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763472/65c66bb2-1ae9-11e7-90a5-16c9217fc0ec.png">

Here's the console:
<img width="1280" alt="screen shot 2017-04-06 at 4 49 24 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763785/49ed0c9c-1aea-11e7-91c6-bcc42512eea4.png">

5. Exited the styling interface and created a box and a pin. The thumbnail of the layer in the legend is a blinking broken image.
<img width="1280" alt="screen shot 2017-04-06 at 4 46 50 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763557/9abe6ae0-1ae9-11e7-91ea-11e800197ba5.png">

6. Successfully created a box and a pin. My layer disappeared on the map but it's still listed on the storylayers list. Saved the mapstory "successfully". Didn't get a 500 error.
<img width="1280" alt="screen shot 2017-04-06 at 4 46 58 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763623/cd4ff0be-1ae9-11e7-90f7-d21bd2b8caaa.png">

7. Published the mapstory "successfully".
<img width="1280" alt="screen shot 2017-04-06 at 4 47 07 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763653/e435c1aa-1ae9-11e7-98ad-758446f57c9a.png">

8. Played my published story. Chapter, box and pin components are there but not the storylayer.
<img width="1280" alt="screen shot 2017-04-06 at 4 47 28 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763682/ff3e8838-1ae9-11e7-8af8-344e25b4ee18.png">

9. Created a new story called 0406h Cables. Tried adding the same storylayer. Thumbnail appears broken in the mini-explorer window.
<img width="1280" alt="screen shot 2017-04-06 at 4 48 30 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763731/285b0dcc-1aea-11e7-88d2-7731e0baf0a3.png">

10. Clicked Use. and failed to load the layer in the composer.
<img width="1280" alt="screen shot 2017-04-06 at 4 48 34 pm" src="https://cloud.githubusercontent.com/assets/4611843/24763765/3a7236b6-1aea-11e7-9926-5524c8d17cfa.png">

**Comment by jonpmarino** [comment]
thanks Rey. This is the same workflow and issue that Ali Jafri experienced from the RFT session on Tuesday (using mapstory.org prod) when he built this story, https://mapstory.org/story/24210. He "broke" the USSR_NATO_EEU layer that he used in this story when changing style and publishing, https://mapstory.org/layers/geonode:ussr_nato_eeu.

**Comment by trepagnier** [comment]
This issue has been temporarily resolved with this commit to Geonode master: https://github.com/MapStory/geonode/commit/5f0c1894a710f02a05f4f980b78cc294c6cdda80

I investigated how we could resolve the issue on the client side: 

TLDR - don't save remote layer configs to the server and load them by default for each composer session.

The problem appears when Geonode attempts to reference the `ows_url` attribute of a base map's layer config, which doesn't exist. In MapLoom, the layer configs for all available remote base maps are being saved to the server inside of each chapter config when a user chooses to save their story. On load, these layer configs are referenced in order to load the base maps. Removing the base map layer configs altogether results in no base maps being loaded. We shouldn't have to store base map information at all except for a reference to what base map the user has chosen. Instead, we could add remote base map layer configs by default to each chapter config, and prevent their configs from being written to the server on save. 

Because this will require a significant refactor of MapLoom, which we'll soon be replacing, I recommend keeping our temporary GeoNode hack in place until we're able to address this in our new composer client.

**Comment by Coop56** [comment]
@darinacosta We'll have to move the hack to the mapstory repo instead of the geonode one then since @sarasafavi will soon be pointing us at core geonode.
